### PR TITLE
New Custom Analytics

### PR DESCRIPTION
**Pull request by txhaflaire**

### CODE REVIEW

**Review by mattbenyo — CHANGES_REQUESTED** [review]
These are great. They held up in testing. I do have one request. Analytics are pretty obtuse, especially for newcomers. One thing that helps demystify them is to break out each logical statement onto a separate line. It's technically just a style choices, but it improves readability and makes everything a hair more user friendly. 

Example:
`$event.type == 1 AND 
$event.process.signingInfo.appid == "com.apple.pfctl" AND 
($event.process.commandLine CONTAINS "-F all" OR 
$event.process.commandLine CONTAINS "-d" OR 
$event.process.commandLine CONTAINS "-e -f" OR 
$event.process.commandLine CONTAINS "-e") OR 
$file.path == "/etc/pf.conf" AND 
$file.isModified == 1`

The UI was also recently updated so that these will hold in the UI if they are pasted in that way. (working on getting the built in ones to display this way. we have them on separate lines in the YAML files)

File: custom_analytic_detections/sudoers_file_modification [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# sudoers_file_modification
```
Comment (mattbenyo) [inline custom_analytic_detections/sudoers_file_modification:1]: This is one that we have had on hold. Ideally, we would be able to show the modification that was made. Unfortunately, as im sure you know, the snapshots aren't working for this kind of file, so we have been holding off until that is up and running again. 

It's a clean detection though, so I don't see any issues putting this on the public repo. We would probably just take it down if/when the snapshots are working and we have it in prod.

Comment (txhaflaire) [reply custom_analytic_detections/sudoers_file_modification:1]: Makes sense, let's put this one on the hold indeed, ideally i wanted to Alert on NOPASSWD entry being added to the file so we could include that once snapshots is up and running.

**Comment by txhaflaire** [comment]
requested re-view after changes to @mattbenyo

**Review by mattbenyo — APPROVED** [review]

**Pull request merged**